jsrelay/docker-compose.yml

```diff
@@ -20,12 +20,12 @@ services:
     volumes:
       - ./gsndata:/app/data    #can be left out, to keep private-key inside the docker
     environment:
-      Url: https://${HOST}
-      Port: 80
-      EthereumNodeUrl: ${NODE_URL}
-      RelayHubAddress: ${RELAY_HUB}
-      GasPricePercent: ${GAS_PRICE_PERCENT}
-      BaseFee: ${BASE_FEE}
-      PercentFee: ${PERCENT_FEE}
+      url: https://${HOST}
+      port: 80
+      ethereumNodeUrl: ${NODE_URL}
```

Comment:
the whole point of config file is use a config file. why still pass everything as cmdline args?
maybe url, workdir still "deserve"  cmdline, but the rest aren't.

Reply:
This should be fixed in a PR after also testing docker imo